modules/clightning/module.cpp

```diff
@@ -92,5 +183,10 @@ namespace bitcoinfuzz
         {
             return clightning_des_invoice(str.c_str());
         }
+
+        std::optional<std::string> CLightning::deserialize_offer(std::string str) const
+        {
+            return clightning_des_offer(str.c_str());
```

Comment:
`c_str()` looks unnecessary here and probably causes 1 or 2 extra copies of `str` to be made  (first to add null byte, then again to convert back to `std::string`).
```suggestion
            return clightning_des_offer(str);
```

Reply:
Fixed